pkg/adapter/github/adapter.go

```diff
@@ -71,9 +71,6 @@ func (ra *Adapter) handleEvent(payload interface{}, hdr http.Header) error {
 			}
 		})
 	}
-	if ra.client == nil {
-		return fmt.Errorf("failed to create cloudevent client")
-	}
```

Comment:
I think I was expecting this second `if ra.client == nil` to remain, but remove the first `if ra.client == nil` around `ra.initClientOnce.Do()`.

Reply:
Is that what you expected @evankanderson?

Reply:
👍 Makes sense, It's better! I'll update it first :)